PHP grpc extension fails to compile

I got errors compiling the grpc extension on ubuntu 14.04

```
$ pecl install grpc
downloading grpc-1.0.0RC3.tgz ...
Starting to download grpc-1.0.0RC3.tgz (1,910,021 bytes)
...............................................................................done: 1,910,021 bytes
884 source files, building
running: phpize
Configuring for:
PHP Api Version:         20131106
Zend Module Api No:      20131226
Zend Extension Api No:   220131226
building in /tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3
running: /tmp/pear/install/grpc/configure --with-php-config=/home/travis/.phpenv/versions/5.6.24/bin/php-config
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for PHP prefix... /home/travis/.phpenv/versions/5.6.24
checking for PHP includes... -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib
checking for PHP extension directory... /home/travis/.phpenv/versions/5.6.24/lib/php/extensions/no-debug-zts-20131226
checking for PHP installed headers prefix... /home/travis/.phpenv/versions/5.6.24/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.13.5 (ok)
checking for gawk... gawk
checking whether to enable grpc support... yes, shared
checking for ld used by cc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for /usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 1572864
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC
checking if cc PIC flag -fPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
running: make
/bin/bash /tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/libtool --mode=compile cc -Wall -Werror     -Wno-parentheses-equality -Wno-unused-value -std=c11     -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN     -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/install/grpc/src/php/ext/grpc/byte_buffer.
c -o src/php/ext/grpc/byte_buffer.lo
mkdir src/php/ext/grpc/.libs
 cc -Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/install/grpc/src/php/ext/grpc/byte_buffer.c  -fPIC -DPIC -o src/php/ext/grpc/.libs/byte_buffer.o
/bin/bash /tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/libtool --mode=compile cc -Wall -Werror     -Wno-parentheses-equality -Wno-unused-value -std=c11     -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN     -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/install/grpc/src/php/ext/grpc/call.c -o sr
c/php/ext/grpc/call.lo
 cc -Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/install/grpc/src/php/ext/grpc/call.c  -fPIC -DPIC -o src/php/ext/grpc/.libs/call.o
/bin/bash /tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/libtool --mode=compile cc -Wall -Werror     -Wno-parentheses-equality -Wno-unused-value -std=c11     -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN     -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/install/grpc/src/php/ext/grpc/call_credent
ials.c -o src/php/ext/grpc/call_credentials.lo
 cc -Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/install/grpc/src/php/ext/grpc/call_credentials.c  -fPIC -DPIC -o src/php/ext/grpc/.libs/call_credentials.o
/bin/bash /tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/libtool --mode=compile cc -Wall -Werror     -Wno-parentheses-equality -Wno-unused-value -std=c11     -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN     -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/install/grpc/src/php/ext/grpc/channel.c -o
 src/php/ext/grpc/channel.lo
 cc -Wall -Werror -Wno-parentheses-equality -Wno-unused-value -std=c11 -fvisibility=hidden -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN -D_HAS_EXCEPTIONS=0 -DNOMINMAX -I. -I/tmp/pear/install/grpc -DPHP_ATOM_INC -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/include -I/tmp/pear/install/pear-build-travisDKrxKw/grpc-1.0.0RC3/main -I/tmp/pear/install/grpc -I/home/travis/.phpenv/versions/5.6.24/include/php -I/home/travis/.phpenv/versions/5.6.24/include/php/main -I/home/travis/.phpenv/versions/5.6.24/include/php/TSRM -I/home/travis/.phpenv/versions/5.6.24/include/php/Zend -I/home/travis/.phpenv/versions/5.6.24/include/php/ext -I/home/travis/.phpenv/versions/5.6.24/include/php/ext/date/lib -I/tmp/pear/install/grpc/include -I/tmp/pear/install/grpc/src/php/ext/grpc -I/tmp/pear/install/grpc/third_party/boringssl/include -DHAVE_CONFIG_H -g -O2 -c /tmp/pear/install/grpc/src/php/ext/grpc/channel.c  -fPIC -DPIC -o src/php/ext/grpc/.libs/channel.o
/tmp/pear/install/grpc/src/php/ext/grpc/channel.c: In function â€˜php_grpc_read_args_arrayâ€™:
/tmp/pear/install/grpc/src/php/ext/grpc/channel.c:87:26: error: too few arguments to function â€˜zend_throw_exceptionâ€™
                          "array_hash is NULL", 1);
                          ^
In file included from /tmp/pear/install/grpc/src/php/ext/grpc/channel.c:46:0:
/home/travis/.phpenv/versions/5.6.24/include/php/Zend/zend_exceptions.h:43:17: note: declared here
 ZEND_API zval * zend_throw_exception(zend_class_entry *exception_ce, const char *message, long code TSRMLS_DC);
                 ^
/tmp/pear/install/grpc/src/php/ext/grpc/channel.c: At top level:
cc1: error: unrecognized command line option "-Wno-parentheses-equality" [-Werror]
cc1: all warnings being treated as errors
make: *** [src/php/ext/grpc/channel.lo] Error 1
ERROR: `make' failed
```

--- a/src/php/ext/grpc/channel.c
+++ b/src/php/ext/grpc/channel.c
@@ -84,7 +84,7 @@ void php_grpc_read_args_array(zval *args_array,
   array_hash = Z_ARRVAL_P(args_array);
   if (!array_hash) {
     zend_throw_exception(spl_ce_InvalidArgumentException,
-                         "array_hash is NULL", 1);
+                         "array_hash is NULL", 1 TSRMLS_CC);
     return;
   }
   args->num_args = zend_hash_num_elements(array_hash);

